### ISSUE TITLE
Adding possibly to add multiple games.

### ISSUE THREAD

**Issue by LukasMFR** [zardam/numworks-nofrendo#4]
Especially for the N0110 seems according to the official site it has 8MB or RAM

**Comment by NzoSifou** [comment]
Yes ! Creator of this emulator, do it please !
You can create a menu in the app "NES" and select games (and on your website propose multiple or single file(s) to upload :D)

**Comment by taokann** [comment]
Oh yes pleaaaase =)

**Comment by LukasMFR** [comment]
Well guess what, you can do it now using Omega

**Comment by NzoSifou** [comment]
Yeah I've been using Omega for a while now x) But the games aren't saved...

**Comment by LukasMFR** [comment]
For me some are some are not.
